modules/serverless-functions-using-integrated-registry.adoc

```diff
@@ -22,3 +22,5 @@ $ kn func build -r $(oc get route -n openshift-image-registry)
 ----
 $ kn func deploy -r $(oc get route -n openshift-image-registry)
 ----
++
+You can verify that the function deployed successfully by emitting a test event to it.
```

Comment:
My ignorance: Do users know how to emit a test? Do we need a link anywhere to show how to emit?

Reply:
@mburke5678 Good catch. What happened is the just-merged docs on emitting a test now begin literally on the next line after the line I've added:
https://github.com/openshift/openshift-docs/pull/33985
So we're good here.